sdk/messaging/azeventgrid/time_rfc3339.go

```diff
@@ -1,5 +1,3 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Another removed license header.

Reply:
I'm looking at the actual file in the branch and it's not removed there. I think this might just be a PR cosmetics error. 

(if it gets removed after merge I'll add it back)